proto/viam/robot/v1/robot.proto

```diff
@@ -360,3 +367,33 @@ message RestartModuleResponse {}
 message ShutdownRequest {}
 
 message ShutdownResponse {}
+
+message GetMachineStatusRequest {}
+
+message GetMachineStatusResponse {
+  repeated ResourceStatus resources = 1;
+}
+
+message ResourceStatus {
+  enum State {
```

Comment:
[q] Does the defining the `enum` inside of `ResourceStatus` make any difference in the compiled code then if you were to define `enum` outside of the message?

Reply:
It shouldn't based on https://github.com/bufbuild/buf/pull/400

> If the previous and current types are both enums, the enums are checked to see if the (1) the short names are equal (2) the previous enum is a subset of the current enum. A subset is defined as having a subset of the name/number enum values. If the previous is a subset, no failure is produced. The idea here is that this covers if someone just moves where an enum is defined, but still allows values to be added to this enum in the same change, as adding values to an enum is not a breaking change.

However, it's worth noting that this was not the case before: https://github.com/bufbuild/buf/issues/80

Reply:
Cool; thanks for looking into it!